app/survey_config/business_config.py

```diff
@@ -18,6 +19,8 @@ class BusinessSurveyConfig(
     def __post_init__(self):
         super().__post_init__()
 
+        self.base_url = self._stripped_base_url
```

Comment:
i think this should be suffice , no need to do property. i am not sure if it is anti-pattern , i hope not
`self.base_url=self.base_url.replace("/surveys/todo", "")` ?

Reply:
I just thought to keep it as a single thing that can be removed later, plus include the warning in the property.

Reply:
Not anymore, just need to remove `self.base_url = self._stripped_base_url` and the function now. Happy to with majority though if there is one